app/index.js

```diff
@@ -147,11 +182,130 @@ module.exports = class extends yeoman {
         userProps.package = this.package;
 
         for (var i = 0; i < this.files.length; i++) {
-            this.fs.copyTpl(
-                this.templatePath(this.files[i]),
-                this.destinationPath(this.files[i].replace(/src\/main\/java/g, path.join(src, packageFolder))),
-                { userProps: userProps }
-            );
+            var skipFile = false;
+            if (showWsdl2Rest) {
+                skipFile = (testFileForWsdl2RestSkip(this.files[i]) == true);
```

Comment:
is it a limitation of javascript to have to compare for boolean before storing into a boolean?
why not directly:
skipFile = testFileForWsdl2RestSkip(this.files[i]);

Reply:
No, it's not a limitation -- was a design decision. If we're in "wsdl2rest" mode, I want it to check the list to see if we should skip the file. If not, I only want to skip the one file. But by default I don't want to accidentally skip a file.

Reply:
so there is a difference between
`skipFile = (testFileForWsdl2RestSkip(this.files[i]) == true);`
and
`skipFile = testFileForWsdl2RestSkip(this.files[i]);`
?

why it would skip the file by default with the second line?

Reply:
Adjusted the code to try and make it a bit clearer and make the non-wsdl2rest case consistent.